users/userService.js

```diff
@@ -0,0 +1,37 @@
+import { FailedCreatingError } from '../errors/errorTypes/failedCreatingError.js';
+import { InvalidDataError } from '../errors/errorTypes/invalidDataError.js';
+import { InvalidLoginError } from '../errors/errorTypes/InvalidLoginError.js';
+import { userValidate } from './userValidate.js';
+import { FAILED_CREATE, INVALID_DATA, INVALID_LOGIN, USERS } from '../utils/textConstants.js';
+import { jwtCreator } from '../jwt/jwtCreator.js';
+
+export class UserService {
+  constructor ({ userRepository }) {
+    this.userRepository = userRepository;
+  }
+
+  create = async (userName, password) => {
+    const userValidated = userValidate(userName, password);
```

Comment:
here userValidation seems like a better option to me. because its more legible in if statement.